--- data/profile.d/Makefile.am	2026-03-26 21:10:10.162781340 +0800
+++ data/profile.d/Makefile.am	2026-06-14 11:21:14.204493239 +0800
@@ -37,7 +37,8 @@
 	rhvh.conf \
 	rocky.conf \
 	scientific-linux.conf \
-	virtuozzo-linux.conf
+	virtuozzo-linux.conf \
+	socreate.conf
 
 
 MAINTAINERCLEANFILES = Makefile.in
--- /dev/null	2026-06-14 09:31:08.397997577 +0800
+++ data/profile.d/socreate.conf	2026-06-14 11:21:14.128851044 +0800
@@ -0,0 +1,21 @@
+# Anaconda configuration file for Socreate OS Server.
+
+[Profile]
+profile_id = socreate
+base_profile = rhel
+
+[Profile Detection]
+os_id = socreate
+
+[Payload]
+
+[Storage]
+file_system_type = xfs
+default_scheme = LVM
+
+[User Interface]
+custom_stylesheet = /usr/share/anaconda/pixmaps/server/socreate-server.css
--- pyanaconda/core/constants.py	2026-03-26 21:10:10.172613364 +0800
+++ pyanaconda/core/constants.py	2026-06-14 11:21:14.222253583 +0800
@@ -283,7 +283,7 @@
 PAYLOAD_STATUS_CHECKING_SOFTWARE = N_("Checking software dependencies...")
 
 # Window title text
-WINDOW_TITLE_TEXT = N_("Anaconda Installer")
+WINDOW_TITLE_TEXT = N_("Socreate OS Installer")
 
 # Types of time sources.
 TIME_SOURCE_SERVER = "SERVER"
